src/routes/chart/get-panels.ts

```diff
@@ -13,7 +13,7 @@ const kindsResources = ['pinGroups', 'pins'] as const;
 type KindsResources = typeof kindsResources[number];
 
 type Request = {
-  query: Record<KindsHashIds, Array<HashId>>;
+  query: Partial<Record<KindsHashIds, Array<HashId>>>;
```

Comment:
Talked about this on the chat with @Arinono where I voiced my concerns what we would expect if the query is optional. In this case it would mean it would return all panels, that could be fine?

Reply:
I think it will mean that we give back nothing.